ci/run-integration-tests.ps1

```diff
@@ -0,0 +1,19 @@
+param (
+    [Parameter(Mandatory=$true)]
+    [string]$RepoName,
+    [Parameter(Mandatory=$true)]
+    [Hashtable]$Keys
+)
+
+# nightly-publish-main workflow doesn't create the examples package, so
+# install-package.ps1 won't install it and its test dependencies won't get
+# installed. That's why we install it here, as a special case.
+if ($env:GITHUB_JOB -eq "Test") {
```

Comment:
Same as the other comment, let's change to look at the name of the calling script instead of environment, unless there's a better solution you want to suggest.

Reply:
I believe this one is different from the other which we can leave? In that it is not just an optimization, but a required check.

Reply:
Yes, this one's different, but there's an easy way to fix it. Only `nightly-publish-main_test.ps1` calls the `install-package.ps1` script, so we can just do the same thing in `install-package.ps1` without any `if`s, and the behavior won't change.